openpathsampling/analysis/tis_analysis.py

```diff
@@ -353,6 +343,9 @@ def total_crossing_probability(self, storage=None, method="wham", force=False):
         force : bool (False)
             if true, cached results are overwritten
         """
+
+        tcp = dict()
```

Comment:
You're mixing two things. Very intentionally, there's a distinction between the local variable `tcp` and the instance variable `self.tcp`. The first is what this part is about, and is a dictionary. The second is a `LookupFunction`. We could initialize the second to `None`, although I think I use its existence (`hasattr`) as a test for whether analysis has already been performed, so you'd have to hunt down those cases and replace that with testing for `is None`. I'm fine with that approach if you think it's worthwhile.